### ISSUE TITLE
Improper ES6 imports

### ISSUE THREAD

**Issue by goldenratio** [englercj/resource-loader#123]
Hi,

in version 2.2.1, we are getting undefined errors.

![screen shot 2018-12-14 at 14 21 35](https://user-images.githubusercontent.com/3019293/50003338-fb92ea00-ffab-11e8-9827-6105e436bc66.png)

On further investigation, in blob.js
Resource class imported as default, but Resource class is not exported as default. (Same for encodeBinary)

```
import Resource from '../../Resource';
import b64 from '../../b64';
```

should be,
```
import { Resource } from '../../Resource';
import { encodeBinary } from '../../b64';
```

![screen shot 2018-12-14 at 14 21 49](https://user-images.githubusercontent.com/3019293/50003354-064d7f00-ffac-11e8-920f-70f2b0d54595.png)

Note: version  2.2.1, is automatically picked up NPM, all new builds are broken.

**Comment by shako92** [comment]
Same here, pixi stopped working:
<img width="699" alt="image" src="https://user-images.githubusercontent.com/2061153/50011889-37698780-ff8b-11e8-941b-16d59f929ebb.png">

And error in console:
<img width="1444" alt="image" src="https://user-images.githubusercontent.com/2061153/50011902-418b8600-ff8b-11e8-9ae7-3f860b56698e.png">

All build are broken.

**Comment by englercj** [comment]
Sorry about this, should be fixed in v2.2.2 now. Let me know if it isn't.
